ui/src/main/webapp/src/app/components/reports/migration-issues/migration-issues-table.component.ts

```diff
@@ -118,4 +118,33 @@ export class MigrationIssuesTableComponent implements OnInit {
 
         this.sortedIssues = this._sortingService.sort(this.migrationIssues);
     }
+
+    showRule(ruleID: string) {
+        /*
+        This is not working either.
```

Comment:
I have used relative navigation in a few places and it seemed to work.  I wonder what the issue is?